### DISCUSSION TITLE
"Error, could not resolve ..."

### DISCUSSION THREAD

**Discussion by Zaskoda** [vu3th/vue-dapp#111] [Q&A] [answered]
I have an existing dapp built in Vue and want to integrate vue-dapp because my wallet connect code is fairly primitive. However, after importing the package and adding .use(VueDapp) I'm getting a series of errors in the console saying "could not resolve ..." for each wallet connect provider. My expectation is that, if the wallet provider was not available, it would quietly ignore it. 

I'm not yet sure if this is an actual bug or if I did something dumb so I haven't created an issue around it. If I determine it is a bug with certainty, I'll do so. But I wanted to make this post just in case it's something obvious with a quick answer that might otherwise take me a lot of time to discern.

**Comment by Zaskoda** [comment, marked as answer]
And just like that, I feel silly... I was on an older node version from working on another project. Switched to the latest and everything is golden.

**Reply by johnson86tw** [reply]
I think I should figure out the lowest version of nodeJS that satisfies this library
